### ISSUE TITLE
An EditorPlugin build callback failed.

### ISSUE THREAD

**Issue by L2750558108** [godotengine/godot#70836]
### Godot version

3.5.1mono

### System information

windows10

### Issue description

I found another similar issue(https://github.com/godotengine/godot/issues/35897) ,but my situation seems a little different.
Every unsuccessful build is accompanied by this error output
      editor/editor_node.cpp:5532 - An EditorPlugin build callback failed.
For example, this output appears when there is a syntax displayed in MSBuild at build time. I think this is redundant because I already have the MSBuild error message. This error message looks as if something went wrong with the editor. If it must exist, I think it would be better to make it clearer (e.g. "Error : MSBuild Failed"). Please tell me if my idea is wrong.

### Steps to reproduce

Create a project
Add a C# script with any syntax error
Run it
See the output and get this:
    editor/editor_node.cpp:5532 - An EditorPlugin build callback failed.

### Minimal reproduction project

An empty project

**Comment by jamboy200** [comment]
hey i have the a same problem .

**Comment by jamboy200** [comment]
hey 

> ### Godot version
> 3.5.1mono
> 
> ### System information
> windows10
> 
> ### Issue description
> I found another similar issue(#35897) ,but my situation seems a little different. Every unsuccessful build is accompanied by this error output editor/editor_node.cpp:5532 - An EditorPlugin build callback failed. For example, this output appears when there is a syntax error displayed in MSBuild at build time. I think this is strange,because it looks as if something went wrong with the editor. I think it would be better to make it clearer (e.g. "Error : Project Build Failed"). Please tell me if my idea is wrong.
> 
> ### Steps to reproduce
> Create a project Add a C# script with any syntax error Run it See the output and get this: editor/editor_node.cpp:5532 - An EditorPlugin build callback failed.
> 
> ### Minimal reproduction project
> An empty project

hey man , i have the same issue as you . i asked the reddit godot community and they replay a  solution the solution worked for me i will give you a link to the post . but the short answer is ifyou are a windows user "C:\Users\username\AppData\Roaming\Godot\mono" go to this directory and if you do not see the a folder named GodotNuGetFallbackFolder you will have to created .
here is the link https://www.reddit.com/r/godot/comments/10gug1u/godot_351_failed_to_build/  

Note : if that solution does not work for you you may want to check the log file in  the directory C:\Users\username\AppData\Roaming\Godot\mono\build_logs for more detail .

and you could post the log file for more info for the community to help you .

hope that was helpfull.

**Comment by L2750558108** [comment]
No. I checked and found that this directory exists, but the editor still shows this error. And I think this should be handled automatically by the editor rather than letting the user check the folder.

**Comment by raulsntos** [comment]
So if I understand this issue correctly, the problem is the _"An EditorPlugin build callback failed"_ message that is printed when building a C# project fails. So the C# project **does** contain errors, but your issue is that the message is unintuitive and sounds like an editor error rather than an user one.

If so, I don't think there's much we can do here. As far as I know, `EditorPlugin` doesn't let you customize the message you get when the overridden `_build` method returns `false`.

The `EditorPlugin` method that fails is the C# plugin and it implements the `_build` method to return `false` when building the C# project fails:

https://github.com/godotengine/godot/blob/23318e877890029f35856036b9c4e0bfa09cacc6/modules/mono/editor/GodotTools/GodotTools/GodotSharpEditor.cs#L397-L400

Then, the message is printed by Core:

https://github.com/godotengine/godot/blob/23318e877890029f35856036b9c4e0bfa09cacc6/editor/editor_node.cpp#L6467-L6470
